stats.go

```diff
@@ -112,6 +115,51 @@ func startPrometheusStats(l *logrus.Logger, i time.Duration, c *config.C, buildV
 	pr.MustRegister(g)
 	g.Set(1)
 
+	// Export the start time of the current binary execution
+	start_time := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   namespace,
+		Subsystem:   subsystem,
+		Name:        "start_time",
+		Help:        "Time the nebula client started, in unixtime",
+		ConstLabels: prometheus.Labels{},
+	})
+	pr.MustRegister(start_time)
+	start_time.Set(float64(time.Now().Unix()))
+
+	// Export our certificate information as labels on a static gauge
+	for _, ca := range caCert.CAs {
+
+		ca_info := prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "ca_cert_expiry",
+			Help:      "Returns CA cert time expiry in unixtime",
+			ConstLabels: prometheus.Labels{
+				// TODO : fill these in with actual data, not hardcoded fields
+				"name":       ca.Details.Name,
+				"public_key": hex.EncodeToString(ca.Details.PublicKey),
+			},
+		})
+
+		pr.MustRegister(ca_info)
+		ca_info.Set(float64(ca.Details.NotAfter.Unix()))
+
+	}
```

Comment:
I'd love comment on the `CAs` slice - what sort of situations would involve multiple CAs? I'll admit I don't understand fully in what situations you might have multiples here, so this section lines 130-147 might be poorly implemented.

Reply:
A few main reasons:
- Disaster recovery, trusting multiple roots and keeping all but the currently active one offline allows you to more easily rotate to a new CA in the event the active CA is compromised.
- Ability to rotate out an expiring CA by trusting the current one and the new one being transitioned to.
- CAs can be limited in scope (ips, groups, and subnets they can issue for). You can use these properties to build parts of your mesh with different levels of trust, while still allowing some or all machines to communicate in some way.
- Use individual CAs to build islands and only allow those islands to communicate with specific machines.

Reply:
Thanks, this is very helpful context. I'll do some more testing and potentially refactoring to account for this new info.